File: python/text/gemini.py
import os
import google.generativeai as genai

# API-KEYの設定
GOOGLE_API_KEY=os.getenv('GOOGLE_API_KEY')

genai.configure(api_key=GOOGLE_API_KEY)

# モデルの選択
# gemini-1.5-flash, gemini-1.5-pro, gemini-1.0-proから選択
model_name = "gemini-1.5-flash"
prompt = "こんにちは"

model = genai.GenerativeModel(model_name)
response = model.generate_content(prompt)
print(response.text)

File: src/javascript/text/gemini.js
import { GoogleGenerativeAI } from "@google/generative-ai";

const GOOGLE_API_KEY = process.env.GOOGLE_API_KEY;

const main = async () => {
  const genAI = new GoogleGenerativeAI(GOOGLE_API_KEY);
  const model = genAI.getGenerativeModel({ model: "gemini-1.5-pro" });

  const prompt = "こんにちは、芝浦工業大学ShibaLabについて教えてください。";

  const result = await model.generateContent(prompt);

  console.log(result.response.text());
};
main();
